scripts/setup-dev.py

```diff
@@ -0,0 +1,162 @@
+# pylint: disable=invalid-name
+import sys
+import os
+import signal
+import subprocess
+
+children = []
+
+try:
+    owd = os.getcwd()
+
+    d = ""
+    noAuthPath = ""
+
+    print("Running script to setup local Arlo instance...")
+
+    res = input("Do you have an instance of nOAuth configured? [y/N]")
```

Comment:
if you are using Arlo for the first time, I would imagine you don't have any idea what nOAuth is or whether it's better to use our instance or your own. maybe add a small explanation or recommendation?

Reply:
Good call. Will add.